repos/rust-lang/cargo.toml

```diff
@@ -2,13 +2,18 @@ org = "rust-lang"
 name = "cargo"
 description = "The Rust package manager"
 homepage = "https://doc.rust-lang.org/cargo"
-bots = ["bors", "rustbot", "rfcbot", "renovate"]
+bots = ["rustbot", "rfcbot", "renovate"]
 
 [access.teams]
 cargo = "write"
 
 [[branch-protections]]
 pattern = "master"
+ci-checks = ["conclusion"]
+
+[[branch-protections]]
+pattern = "beta"
+ci-checks = ["conclusion"]
```

Comment:
Oh, sorry, to be clear, the pattern isn't "beta". It should be the `rust-1.*` pattern below. That's where we put our beta backports (it works a little differently than rust-lang/rust).

Reply:
Right, sorry, my bad. Fixed :)